lib/core/src/lib/form/components/widgets/amount/amount.widget.spec.ts

```diff
@@ -327,6 +462,315 @@ describe('AmountWidgetComponent - rendering', () => {
             expect(asterisk.textContent).toEqual('*');
         });
     });
+
+    describe('Test widget with ADF_AMOUNT_SETTINGS as observable', () => {
+        beforeEach(() => {
+            TestBed.resetTestingModule();
```

Comment:
should not be required as entire test suite is configured to do that, see https://github.com/Alfresco/alfresco-ng2-components/blob/b16e326ac34b72d8b55f46ded412881529856298/lib/core/test.ts#L26

Reply:
I need to do it because below I am defining testing module for ADF_AMOUNT_SETTINGS as observable.